manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Aragon Network",
+  "description": "Aragon Network Dashboard",
+  "icons": [
+    {
+      "src": "images/icon.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    }
+  ],
+  "start_url": "index.html"
```

Comment:
I don’t think this file gets copied in `public/` during the build, should we add this to `scripts/build`?